pkg/lvm/lvm_util.go

```diff
@@ -53,13 +53,15 @@ const (
 	LVCreate = "lvcreate"
 	LVRemove = "lvremove"
 	LVExtend = "lvextend"
+	LVChange = "lvchange"
 	LVList   = "lvs"
 
 	PVList = "pvs"
 	PVScan = "pvscan"
 
 	YES        = "yes"
 	LVThinPool = "thin-pool"
+	SUDO       = "sudo"
```

Comment:
Was sudo not required previously?

Reply:
It usually runs from privileged container so `sudo` isn't required.  I added that so the ci script can also run this. However, I've reworked this a bit and there are no more changes required in `lvm_util.go` . Updated diff in PR 308. The commit 2ca22d9 in this PR is from PR 308 since this PR's branch is on top of that.